atom/browser/browser.h

```diff
@@ -216,6 +223,8 @@ class Browser : public WindowListObserver {
   std::string version_override_;
   std::string name_override_;
 
+  int current_badge_count_ = 0;
```

Comment:
I guess the compiler would remove this as unused, because it is private.
But I will add an ifdef block, to keep the code clean.